installer/installation-notes.md

```diff
@@ -0,0 +1,135 @@
+# Installing the RSP
+
+## Preparation
+
+  * Create a working dir.
+  * git clone https://github.com/lsst-sqre/phalanx into it (using https
+    rather than git+ssh is important).
+  * cd phalanx/installer
+
+## Create installation environment
+
+  * If you don't have a virtualenv for the installer already,
+    `mkvirtualenv -p $(which python3) -r requirements.txt installer`
+    (adjust if you use pyenv or raw virtualenv or whatever; the
+    important thing is to pick up the requirements.txt from the
+    installer directory)
+  * Get the correct write token for this instance's Vault enclave out of
+    1Password
+
+## Prepare for the ingress
+
+There are three cases (thus far) for what the top-level ingress looks
+like.
+
+  * It is externally managed, as are any required TLS certificates.
+    The installer doesn't cope with this case very well yet, and you
+    will need to manually inject the TLS certificate into the squareone
+    configuration post-installation.
```

Comment:
I don't understand -- if the TLS certificate is externally managed, why do you have to inject it anywhere? For NCSA, I don't think we reference the TLS certificate in any way.